packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -333,12 +332,24 @@ class _RenderableTileLayer extends _RenderableLayer<TileLayer> {
   }
 
   void _cacheLayerTiles() {
-    if (_map.orientation == MapOrientation.isometric) {
-      _cacheIsometricTiles();
-    } else if (_map.orientation == MapOrientation.hexagonal) {
-      _cacheHexagonalTiles();
-    } else {
-      _cacheOrthogonalLayerTiles();
+    for (final batch in _cachedSpriteBatches.values) {
+      batch.clear();
+    }
+
+    switch (_map.orientation) {
+      case MapOrientation.isometric:
+        _cacheIsometricTiles();
+        break;
+      case MapOrientation.staggered:
+        _cacheIsometricStaggeredTiles();
+        break;
+      case MapOrientation.hexagonal:
+        _cacheHexagonalTiles();
+        break;
+      case MapOrientation.orthogonal:
+      default:
```

Comment:
If you exhaust the enum options here you don't need the default case. It's actually probably better to remove it, because then if you add another value to the enum then the analyzer will complain here that it doesn't check all values, like it should.

Reply:
will do